open-apparel-registry/open-apparel-registry
# Upgrade Dependencies

## Overview

Upgrades Redux, React-Router and React-Select to the latest versions.
The outdated versions were introducing warnings into the console due
to not being optimized for React v16.

Connects #1300 

## Demo

Manageable level of console warnings:
<img width="931" alt="Screen Shot 2021-04-05 at 12 37 17 PM" src="https://user-images.githubusercontent.com/21046714/113599029-07d36300-960c-11eb-8f4e-850e93f4f4d0.png">

## Testing Instructions

* Run `./scripts/update` and `./scripts/server`
* Confirm the website works as before
* Open the developer console and confirm there are no warnings about UNSAFE_ lifecycle models. 

## Checklist
